recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
 {% set name = "python-suitesparse-graphblas" %}
-{% set version = "8.2.1.0" %}
+{% set version = "9.3.1.0" %}
```

Comment:
Just so I understand, this updating the version as well?

Reply:
Yes, 9.3.1.0 is the first version that we specifically updated to support NumPy 2